suite-common/wallet-core/src/transactions/__fixtures__/transactionsReducer.ts

```diff
@@ -149,3 +149,30 @@ export const addTransaction = [
         result: { [ACCOUNT.key]: [testMocks.getWalletTransaction({ txid: '00' })] },
     },
 ];
+
+export const removeTransaction = [
+    {
+        description: 'ordinary remove',
```

Comment:
I suggest writing test case description is a way that explaining what is the expected behavior and why we need this this way.

In this case something like: `it("removes the transaction from the existing account", ...`